### PR TITLE
chore: [PAGOPA-1787] Update upload report mapping

### PR DESCRIPTION
**Pull request by AngeloCaporaso**
#### List of Changes

- Get report retrieve if doesn't exists as Status on cosmos-db
- Fix performance test
- Set `PaymentPosition` and `IUPDs` `minSize` to 1 (avoid files upload with zero PP)

#### Motivation and Context

- Allow the retrieve of the report after it has been deleted as Status from the database (Status time to live)

CosmosPatchOperation application can be found here:
- https://github.com/pagopa/pagopa-gpd-upload-function/pull/20
- https://github.com/pagopa/pagopa-gpd-upload-function/pull/22

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/code_review.yml [hunk 1]
```diff
@@ -47,6 +47,7 @@ jobs:
   smoke-test:
     name: Smoke Test
     runs-on: ubuntu-latest
+    if: false # todo
```
Comment (FedericoRuzzier) [inline .github/workflows/code_review.yml:50]: Remove or complete todo?

File: performance-test/src/main_scenario.js [hunk 2]
```diff
@@ -61,27 +61,43 @@ export default function () {
...
+  let checkOutput = check(r, { 'check file upload response is 202': (_r) => r.status === 202, }, { tag: "File Upload API" });
 
-  // check(r, { 'check status is 202': (_r) => r.status === 202, }, tag);
+  if (!checkOutput) {
+    console.log(`status: ${r.status},\nbody: ${r.json.body}\n`)
+  }
 
   let statusURL = r.headers['Location']
-  r = http.get(`${vars.host}/${statusURL}`, params)
+  console.log(`url-upload-key: ${statusURL}`)
```
Comment (FedericoRuzzier) [inline performance-test/src/main_scenario.js:71]: Debug

File: src/main/java/it/gov/pagopa/gpd/upload/repository/BlobStorageRepository.java [hunk 3]
```diff
@@ -127,4 +129,19 @@ private String uploadFileBlocksAsBlockBlob(BlockBlobClient blockBlob, File file)
...
+        BlobContainerClient blobContainerClient = blobServiceClient.getBlobContainerClient(broker);
+        String blobName = filekey.concat(".json");
+
+        if(!blobContainerClient.exists())
+            log.error(String.format("[Error][BlobStorageRepository@getReport] Container doesn't exist: %s, for upload: %s", broker, filekey));
+
+        BlobClient blobClient = blobContainerClient.getBlobClient("/" + fiscalCode + "/" + OUTPUT_DIRECTORY + "/report" + blobName);
+
+        if(Boolean.FALSE.equals(blobClient.exists()))
+            log.error(String.format("[Error][BlobStorageRepository@getReport] Blob doesn't exist: %s", filekey));
```
Comment (FedericoRuzzier) [inline src/main/java/it/gov/pagopa/gpd/upload/repository/BlobStorageRepository.java:143]: Maybe launching an exception?

File: src/main/java/it/gov/pagopa/gpd/upload/repository/StatusRepository.java [hunk 4]
```diff
@@ -64,11 +65,12 @@ public Status findStatusById(String id, String fiscalCode) {
...
         } catch (CosmosException ex) {
-            log.error("[Error][StatusRepository@findStatusById] The Status retrieval was not successful: " + ex.getStatusCode());
+            log.error("[Error][StatusRepository@findStatusById] The Status retrieval was not successful: {}", ex.getStatusCode());
             if(ex.getStatusCode() == HttpStatus.INTERNAL_SERVER_ERROR.getCode())
-                throw new AppException(HttpStatus.INTERNAL_SERVER_ERROR, HttpStatus.INTERNAL_SERVER_ERROR.name(), "Status saving unavailable");
-            else
-                throw new AppException(HttpStatus.valueOf(ex.getStatusCode()), String.valueOf(ex.getStatusCode()), "Status retrieval failed");
+                throw new AppException(HttpStatus.INTERNAL_SERVER_ERROR, HttpStatus.INTERNAL_SERVER_ERROR.name(), "The Status for given fileId " + id + " is not available");
+            else if(ex.getStatusCode() == NOT_FOUND.getCode())
+                throw new AppException(NOT_FOUND, "STATUS NOT FOUND", "The Status for given fileId " + id + " does not exist");
```
Comment (FedericoRuzzier) [inline src/main/java/it/gov/pagopa/gpd/upload/repository/StatusRepository.java:72]: Can we use format string like at 68?

File: .github/workflows/code_review.yml (hunk 1 above)
Comment (AngeloCaporaso) [reply .github/workflows/code_review.yml:50]: `todo` with a task ad-hoc

File: src/main/java/it/gov/pagopa/gpd/upload/repository/BlobStorageRepository.java (hunk 3 above)
Comment (AngeloCaporaso) [reply src/main/java/it/gov/pagopa/gpd/upload/repository/BlobStorageRepository.java:143]: Good idea

File: src/main/java/it/gov/pagopa/gpd/upload/repository/StatusRepository.java (hunk 4 above)
Comment (AngeloCaporaso) [reply src/main/java/it/gov/pagopa/gpd/upload/repository/StatusRepository.java:72]: I will update it with `String.format(...)`

**Review by FedericoRuzzier — APPROVED** [review]
LGTM 👍

**Pull request merged**